components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -546,6 +415,27 @@ impl AnimationValue {
             _ => return None,
         })
     }
+
+    /// Update `style` with the value of this `AnimationValue`.
+    pub fn set_in_style(&self, style: &mut ComputedValues) {
```

Comment:
This is a bit problematic because e.g., it doesn't update `em` units when you animate `font-size`, etc.

It's not worse than what we were doing, but can you add a big warning that this is supposed to be used in Servo only for now?

Reply:
Oh, interesting. Hopefully that's fixed once we properly cascade animation and transitions values.

Reply:
I've added a warning and renamed this to `set_in_style_for_servo` so it's event more obvious.